lib/money/money.rb

```diff
@@ -337,6 +341,9 @@ def initialize( obj, currency = Money.default_currency, options = {})
     @currency ||= Money.default_currency
     @bank       = obj.respond_to?(:bank) ? obj.bank : options[:bank]
     @bank     ||= Money.default_bank
+    @infinite_precision = obj.respond_to?(:infinite_precision?) ? obj.infinite_precision? : options[:infinite_precision]
```

Comment:
This line suggests that `:infinite_precision` option will be ignored when initialized with Money `obj`. I think explicit options should always take precedence. WDYT?

Reply:
@antstorm you're right! I've missed that one.
The thing is - this behavior is copied from the bank setting. Don't you think we should change it for bank as well?

Reply:
Let's probably leave `:bank` for now as it will introduce a breaking change. We can do this in one of the bigger releases